src/tests/baseservices/compilerservices/UnsafeAccessors/UnsafeAccessorsTests.cs

```diff
@@ -373,16 +373,23 @@ public static void Verify_ManagedUnmanagedFunctionPointersDontMatch()
         extern static string CallManagedMethod(UserDataClass d, delegate* unmanaged[Cdecl]<void> fptr);
     }
 
-    class InheritanceBase
+    abstract class InheritanceBase
     {
         private static string OnBase() => nameof(OnBase);
         private static string FieldOnBase = nameof(FieldOnBase);
+        protected abstract string Abstract();
+        protected virtual string Virtual() => $"{nameof(InheritanceBase)}.{nameof(Virtual)}";
+        protected virtual string NewVirtual() => $"{nameof(InheritanceBase)}.{nameof(NewVirtual)}";
+        protected virtual string BaseVirtual() => $"{nameof(InheritanceBase)}.{nameof(BaseVirtual)}";
     }
 
     class InheritanceDerived : InheritanceBase
     {
         private static string OnDerived() => nameof(OnDerived);
         private static string FieldOnDerived = nameof(FieldOnDerived);
+        protected override string Abstract() => $"{nameof(InheritanceDerived)}.{nameof(Abstract)}";
```

Comment:
Do we have any concerns around the "we can ignore modopts unless there's ambiguity" interfering with "find methods on base type if current type doesn't have them" that would be worth testing?

Reply:
As in a base type has a function that has modopts where the derived type doesn't?

Reply:
Yeah, something like this:

```csharp
using System;
using System.Runtime.CompilerServices;

int x = 123;
Console.WriteLine(Method(new Derived(), ref x));

[UnsafeAccessor(UnsafeAccessorKind.Method, Name = nameof(Method))]
extern static string Method(Derived target, ref int x);

class Base
{
    public virtual string Method(ref int x) => "ref";
}

class Derived : Base
{
    public virtual string Method(in int x) => "in";
}
```

(This is matching `Method` on `Derived` because there was a conflict when ignoring modopts.)